packages/atomic/cypress/integration/facets/category-facet.cypress.ts

```diff
@@ -0,0 +1,456 @@
+        expect(analyticsBody).to.have.property('actionCause', 'facetSelect');
+        expect(analyticsBody.customData).to.have.property(
+          'facetValue',
+          canadaHierarchy[0]
+        );
+        expect(analyticsBody.customData).to.have.property(
+          'facetField',
+          categoryFacetProp.field
+        );
+        expect(analyticsBody.facetState[0]).to.have.property(
+          'state',
+          'selected'
+        );
+        expect(analyticsBody.facetState[0]).to.have.property(
+          'field',
+          categoryFacetProp.field
+        );
+        expect(analyticsBody.facetState[0]).to.have.property(
+          'facetType',
+          'hierarchical'
+        );
+      });
+    });
+
+    it('Should trigger breadcrumb and display correctly', () => {
+      cy.get(BreadcrumbSelectors.breadcrumb)
+        .shadow()
+        .find('div[part="breadcrumb-wrapper"]')
+        .find('button[part="breadcrumb-button"]')
+        .should('be.visible')
+        .contains(canadaHierarchy[0]);
+    });
+
+    describe('When user click on "All Categories" button', () => {
+      beforeEach(() => {
+        cy.get('@categoryClearAllButton').click();
+        cy.wait('@coveoAnalytics');
+      });
+
+      it('Should go back to the first parent level', () => {
+        cy.get('@categoryClearAllButton').should('not.exist');
+        assertNonZeroFacetCount();
+      });
+
+      it('Should log UA correctly', () => {
+        cy.wait('@coveoAnalytics').then((intercept: any) => {
+          const analyticsBody = intercept.request.body;
+          expect(analyticsBody).to.have.property(
+            'actionCause',
+            'facetClearAll'
+          );
+          expect(analyticsBody.customData).to.have.property(
+            'facetField',
+            categoryFacetProp.field
+          );
+        });
+      });
+    });
+
+    describe('When user clicks ShowMore button', () => {
+      it('Should expand list of childrens on same level', () => {
+        assertClickShowMore(categoryFacetProp.numberOfValue);
+      });
+
+      it('Should log UA correctly', () => {
+        cy.wait('@coveoAnalytics');
+        assertShowMoreUA(categoryFacetProp.field);
+      });
+
+      describe('When user click ShowLess button', () => {
+        it('Should collapse the list and hide ShowLess button', () => {
+          assertClickShowLess(categoryFacetProp.numberOfValue * 2);
+        });
+
+        it('Should log UA correctly', () => {
+          cy.wait('@coveoAnalytics');
+          assertShowLessUA(categoryFacetProp.field);
+        });
+      });
+    });
+  });
+
+  describe('When user goes to last child on level 3', () => {
+    beforeEach(() => {
+      clickOnCategoryFacetWithValue(canadaHierarchy[0]);
+      cy.wait('@coveoAnalytics');
+      clickOnCategoryFacetWithValue(canadaHierarchy[1]);
+      cy.wait('@coveoAnalytics');
+      clickOnCategoryFacetWithValue(canadaHierarchy[2]);
+      cy.wait(500);
```

Comment:
why the 500?

Reply:
just in case it clicks too fast. I will change to `cy.wait('@coveoSearch')`